components/sections/Skills.tsx

```diff
@@ -14,7 +14,7 @@ const SkillsSection = () => {
   }, []);
 
   return (
-    <section className='w-full'>
+    <section className='w-full' data-cy='skills'>
```

Comment:
It should be consistent follow either `className then data` / `data then className`